src/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export default function Header({ appModes, setAppModehandler }) {
```

Comment:
Just a small nitpick the instructions call for a component named 'Heading', while you have called it 'Header'. I'd make sure you follow the instructions as close as possible when they are so detailed, as this is often an important evaluation criterion for interviews.